charts/backbeat/templates/lifecycle/conductor_deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "backbeat.fullname" . }}-lifecycle-conductor
+  labels:
+    app: {{ template "backbeat.name" . }}-lifecycle
+    chart: {{ template "backbeat.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "backbeat.name" . }}-lifecycle
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: conductor
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/usr/src/app/docker-entrypoint.sh"]
+          args: ["npm", "run", "lifecycle_conductor"]
+          env:
+            - name: ZOOKEEPER_AUTO_CREATE_NAMESPACE
+              value: "1"
+            - name: ZOOKEEPER_CONNECTION_STRING
+              value: "{{- printf "%s-zenko-quorum:2181" .Release.Name | trunc 63 | trimSuffix "-" -}}"
+            - name: KAFKA_HOSTS
+              value: "{{- printf "%s-zenko-queue:9092" .Release.Name .Release.Name | trunc 63 | trimSuffix "-" -}}"
+            - name: AUTH_TYPE
+              value: service
+            - name: AUTH_ACCOUNT
+              value: service-lifecycle
+            - name: EXTENSIONS_LIFECYCLE_ZOOKEEPER_PATH
+              value: "{{ .Values.lifecycle.zookeeper.path }}"
+            - name: EXTENSIONS_LIFECYCLE_BACKLOG_METRICS_ZKPATH
+              value: "{{ .Values.lifecycle.zookeeper.backlog_metrics.path }}"
+            - name: EXTENSIONS_LIFECYCLE_BACKLOG_METRICS_INTERVALS
+              value: "{{ .Values.lifecycle.zookeeper.backlog_metrics.interval }}"
+            - name: EXTENSIONS_LIFECYCLE_BUCKET_TASK_TOPIC
+              value: bucket_tasks_topic
```

Comment:
The actual topic name we use is `backbeat-lifecycle-bucket-tasks`, should be set here and for other lifecycle components.